File: app/src/main/java/com/karigor/friends/model/Location.java
package com.karigor.friends.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Location  implements Serializable {

    @SerializedName("city")
    @Expose
    private String city;
    @SerializedName("state")
    @Expose
    private String state;
    @SerializedName("country")
    @Expose
    private String country;

    public String getCity() {
        return city;
    }

    public String getState() {
        return state;
    }

    public String getCountry() {
        return country;
    }
}

File: app/src/main/java/com/karigor/friends/model/Location.kt
package com.karigor.friends.model

import com.google.gson.annotations.SerializedName
import com.google.gson.annotations.Expose
import java.io.Serializable

class Location : Serializable {
    @SerializedName("city")
    @Expose
    val city: String? = null

    @SerializedName("state")
    @Expose
    val state: String? = null

    @SerializedName("street")
    @Expose
    val street: Street? = null

    @SerializedName("country")
    @Expose
    val country: String? = null
}